app/views/spree/admin/orders/invoice4.html.haml

```diff
@@ -55,6 +55,9 @@
         %br
         = t :order_number
         = @order.number
+        %br
+        = "#{t :order_cycle} :"
```

Comment:
I notice that some fields have the colon (`:`) included in the translation file, and others have it in the template here. Also there's usually no space before the colon.

I'm not sure what the best way is, but it would be good to choose one way. I notice that most fields have the colon included in the translation file, so that would be my suggestion (IE, same as order_number above)
```suggestion
        = t :order_cycle
```